redbot/cogs/mod/settings.py

```diff
@@ -199,3 +204,19 @@ class ModSettings(MixinMeta):
                     command=f"{ctx.prefix}unban"
                 )
             )
+
+    @modset.command()
+    @commands.guild_only()
+    async def toggledm(self, ctx: commands.Context, enabled: bool = None):
+        """Toggle whether to send a message to a user when they are kicked/banned.
+        If this is True, the bot will attempt to DM the user with the guild name
+        and reason as to why they were kicked/banned.
+        """
+        guild = ctx.guild
+        if enabled is None:
+            enabled = not await self.settings.guild(guild).toggle_dm()
+        await self.settings.guild(guild).toggle_dm.set(enabled)
+        if enabled:
+            await ctx.send(_("Users will now be DM'd when they are kicked/banned."))
+        else:
+            await ctx.send(_("Users will not be DM'd when they are kicked/banned"))
```

Comment:
DM'd is not a word, we should use `User will not recieve a DM when they are kicked/banned.`